### ISSUE TITLE
issues running preprocessing: fixed_zero_mean_unit_variance

### ISSUE THREAD

**Issue by pr4deepr** [deepimagej/deepimagej-plugin#53]
Hi
   I am planning to release a better model than the one I referenced in #52 . 
Considering I'm having trouble with the old model in recent deepimagej, I thought I'd fast track this, but I'm running into troubles..
This is a pytorch model and it was running on a previous version of deepimagej, however it was using the macros for pre and post-processing. 

As the new version of deepimageJ does not use macros, when I used `fixed_zero_mean_unit_variance` for preprocessing it throws an error. 

```
[SERVICE-1] {"responseType":"LAUNCH","task":"12250151-c846-49dc-a16c-d861d075e897"}
[WORKER-1] SLF4J: No SLF4J providers were found.
[WORKER-1] SLF4J: Defaulting to no-operation (NOP) logger implementation
[WORKER-1] SLF4J: See https://www.slf4j.org/codes.html#noProviders for further details.
[SERVICE-1] 
[SERVICE-1] <INVALID> 
[SERVICE-1] <INVALID> Loading:     100% |========================================|
[SERVICE-1] {"responseType":"COMPLETION","task":"12250151-c846-49dc-a16c-d861d075e897"}
java.lang.IllegalArgumentException: Invalid method 'fixed_zero_mean_unit_variance' in the specs file. The method does not exist in the Bioimage.io framework.
	at io.bioimage.modelrunner.model.processing.TransformationInstance.findClassInClassPath(TransformationInstance.java:193)
	at io.bioimage.modelrunner.model.processing.TransformationInstance.getTransformationClass(TransformationInstance.java:177)
	at io.bioimage.modelrunner.model.processing.TransformationInstance.build(TransformationInstance.java:166)
	at io.bioimage.modelrunner.model.processing.TransformationInstance.<init>(TransformationInstance.java:63)
	at io.bioimage.modelrunner.model.processing.TransformationInstance.create(TransformationInstance.java:78)
	at io.bioimage.modelrunner.model.processing.Processing.buildPreprocessing(Processing.java:79)
	at io.bioimage.modelrunner.model.processing.Processing.<init>(Processing.java:66)
	at io.bioimage.modelrunner.model.processing.Processing.init(Processing.java:110)
	at io.bioimage.modelrunner.model.Model.runBMZ(Model.java:688)
	at io.bioimage.modelrunner.model.Model.runBMZ(Model.java:614)
	at io.bioimage.modelrunner.model.Model.runBMZ(Model.java:575)
	at deepimagej.Runner.runOnTestImages(Runner.java:146)
	at deepimagej.gui.Gui.lambda$runModelOnTestImage$23(Gui.java:314)
	at java.lang.Thread.run(Thread.java:75
```

You can download the [model from this onedrive link](https://wehieduau-my.sharepoint.com/:u:/g/personal/rajasekhar_p_wehi_edu_au/EQdX7q2gEaxMuw-ll8i5tx4B0NzUi38j_jIb7mqYMMWA5Q?e=Rkslmc)

I think I may need help fixing up the rdf yaml file.. 

This is the output I expect from the sample image:

![image](https://github.com/user-attachments/assets/d00ccb48-88f7-4535-b230-d5a5628f46bb)

-------
Something else I saw:

The test-input.npy appears to open as a rotated and flipped image when I click `Run on Test Image`

The npy file in Python:
![image](https://github.com/user-attachments/assets/efa316f0-320a-43d1-a372-a7ceaa601c6b)

When I click on Run on test in deepimagej, the image opens like this:

![image](https://github.com/user-attachments/assets/8d1cf0a0-e076-4236-b599-88f2445e3c93)

Cheers
Pradeep

**Comment by pr4deepr** [comment]
Hi @carlosuc3m 
  I realized I omitted a normalization step in the preprocessing. 

In Python, I preprocess the image by:

- dividing image by 255
- normalize by mean and std, i.e., zero_mean_unit_variance

**How do I specify two preprocessing steps in the rdf.yaml?**

The dividing image by 255, seems to be the `scale_range` preprocessing. with min_percentile of 0 and max_percentile of 100.

Cheers
Pradeep
